epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityItem.java

```diff
@@ -27,24 +27,30 @@
   private int adapterPosition = RecyclerView.NO_POSITION;
 
   @Px
-  private int sizeInScrollingDirection;
-
-  private int sizeNotInScrollingDirection;
-
-  private boolean verticalScrolling;
+  private int height;
+  @Px
+  private int width;
 
-  private float percentVisibleSize = 0.f;
+  private float percentVisibleHeight = 0.f;
```

Comment:
would it be possible to compute this as needed from `visibleHeight` and `height`, instead of storing it as a field?

seems more fragile and complex to keep things in sync if we store fields we could derive